app/models/auth_source.rb

```diff
@@ -31,11 +32,10 @@ class AuthSource < ApplicationRecord
   scoped_search :relation => :locations, :on => :id, :rename => :location_id, :complete_enabled => false, :only_explicit => true, :validator => ScopedSearch::Validators::INTEGER
   scoped_search :relation => :organizations, :on => :name, :rename => :organization, :complete_value => true, :only_explicit => true
   scoped_search :relation => :organizations, :on => :id, :rename => :organization_id, :complete_enabled => false, :only_explicit => true, :validator => ScopedSearch::Validators::INTEGER
+  audit_associations :organizations, :locations
```

Comment:
I think we don't need this here, the child classes that can actually have taxonomies already include taxonomix

Reply:
yeah that makes sense, will remove

Reply:
ah that was actually required, Audited uses class variable that is sadly shared for all classes in STI tree, therefore if some child class adds Taxonomix, loading other child class will fail

I'm working on fix in audited meanwhile, but I've heard there's not gonna be release anytime soon, so this handles the problem here. I've added comment.